Packs/PrismaCloud/Integrations/PrismaCloudIAM/PrismaCloudIAM.py

```diff
@@ -0,0 +1,375 @@
+
+        :return: An IAMUserAppData object that contains the data of the updated user in the application.
+        :rtype: ``IAMUserAppData``
+        """
+        uri = f'user/{user_id}'
+        self._http_request(
+            method='PUT',
+            url_suffix=uri,
+            json_data=user_data,
+            return_empty_response=True,
+            empty_valid_codes=[200]
+        )
+        return IAMUserAppData(user_id=None, username=None, is_active=None, app_data=user_data)
+
+    def enable_user(self, user_id: str) -> IAMUserAppData:
+        """ Enables a user in the application using REST API.
+
+        :type user_id: ``str``
+        :param user_id: ID of the user in the application
+
+        :return: An IAMUserAppData object that contains the data of the user in the application.
+        :rtype: ``IAMUserAppData``
+        """
+        return self.enable_disable_user(user_id=user_id, enable=True)
+
+    def disable_user(self, user_id: str) -> IAMUserAppData:
+        """ Disables a user in the application using REST API.
+
+        :type user_id: ``str``
+        :param user_id: ID of the user in the application
+
+        :return: An IAMUserAppData object that contains the data of the user in the application.
+        :rtype: ``IAMUserAppData``
+        """
+        return self.enable_disable_user(user_id=user_id, enable=False)
+
+    def enable_disable_user(self, user_id: str, enable: bool) -> IAMUserAppData:
+        """ Enables or disable a user in the application using REST API.
+
+        :type user_id: ``str``
+        :param user_id: ID of the user in the application
+
+        :type enable: ``bool``
+        :param enable: True to enable, False for disable
+
+        :return: An IAMUserAppData object that contains the data of the user in the application.
+        :rtype: ``IAMUserAppData``
+        """
+
+        status = 'true' if enable else 'false'
+        uri = f'user/{user_id}/status/{status}'
+        self._http_request(
+            method='PATCH',
+            url_suffix=uri,
+            return_empty_response=True,
+            empty_valid_codes=[200]
+        )
+        return IAMUserAppData(user_id=None, username=None, is_active=enable, app_data=None)
+
+    def get_app_fields(self) -> Dict[str, Any]:
+        """ Gets a dictionary of the user schema fields in the application and their description.
+
+        :return: The user schema fields dictionary
+        :rtype: ``Dict[str, str]``
+        """
+
+        return USER_SCHEMA
+
+    def create_login(self):
+        try:
+            uri = '/login/'
+            body = {
+                "username": self._username,
+                "password": self._password,
+                "customerName": self._customer_name
+            }
+            return self._http_request(
+                method='POST',
+                url_suffix=uri,
+                json_data=body
+            )
+        except DemistoException as e:
+            return_error(e.message)
```

Comment:
It's a better practice to catch the exceptions in the main function.
Look at how `main` is implemented in SalesForce IAM for example.

Reply:
I will change the implementation 
but still will need a separated catch for test_module fail in order to display correct message